client/devdocs/gutenberg-blocks/examples.jsx

```diff
@@ -0,0 +1,239 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+export default [
+	{
+		name: 'core/audio',
+		attributes: {
+			autoplay: false,
+			caption: 'Cartoon Birds',
+			loop: false,
+			preload: 'none',
+			src: 'http://soundbible.com/mp3/cartoon-birds-2_daniel-simion.mp3',
+		},
```

Comment:
Have we considered providing an example block or properties definition file from the package itself? It feels like this will get out of sync eventually.

Reply:
One other though might be to scan the package, but I'm not sure how well formed that would be.

Reply:
it shouldn't be very complicated. there are already fixtures that could be re-used: https://github.com/WordPress/gutenberg/blob/72da0cd7c358c5ffda63305e45183c0dcf3c21a7/test/integration/full-content/fixtures/core__audio.json

Reply:
we could also include a README file for each block, so we can display some docs when viewing an individual block in Calypso

cc @gziolo @youknowriad